doc/upgrade/7.2.md

```diff
@@ -0,0 +1,28 @@
+# Upgrade steps from 7.1 to 7.2
+
+See `doc/bc/changes-7.2.md` for requirements changes and deprecations.
+
+## MySQL/MariaDB database tables character set change
+
+The character set for MySQL/MariaDB database tables is changed from `utf8` to `utf8mb4` to support 4-byte characters. See `data/update/mysql/dbupdate-7.1.0-to-7.2.0.sql` for the SQL upgrade statements.
+
+Beware that these upgrade statements may fail due to index collisions. This is because the indexes have been shortened, so duplicates may occur. If that happens, you must remove the duplicates manually, and then repeat the statements that failed.
+
+You should also change the character set that is specified in the application config, and in legacy if you use that.
+
+In `app/config/config.yml`, set the following:
+```yml
+doctrine:
+    dbal:
+        connections:
+            default:
+                charset: utf8mb4
+```
+Also make the corresponding change in `app/config/dfs/dfs.yml`.
+
+For legacy, in `ezpublish_legacy/settings/i18n.ini`, set the following:
```

Comment:
In my case, this results in heavy errors in legacy. I think it has to do with the fact that legacy is unable to read the INI files now, every call to `$ini->variable()` returns `false`. Any tips @glye @andrerom ?

Reply:
@emodric That Should Not Happen™ and didn't for me. Are you saying it expects the ini files to have `utf8mb4` charset, and returns nothing because that charset doesn't exist in this context? Disturbing news :(

Reply:
I'm not 100% sure that's the case, but it does sound like it. The only outcome that I witnessed is that reading any INI variable just returns `false`.

Reply:
@emodric I did it for reasons that elude me know. It does look like you're right.